install.sh

```diff
@@ -564,30 +570,255 @@ compare_versions() {
   return 0
 }
 
-get_linux_distro() {
-  declare distro
-  if [[ -f /etc/os-release ]]; then
-    . "/etc/os-release"
-    distro="$NAME"
-  elif type lsb_release 1>/dev/null 2>&1; then
-    distro="$(lsb_release -si)"
-  elif [[ -f "/etc/lsb-release" ]]; then
-    . "/etc/lsb-release"
-    distro="$DISTRIB_ID"
-  elif [[ -f "/etc/debian_version" ]]; then
-    # Older Debian/Ubuntu/etc.
-    distro="debian"
-  elif [[ -f "/etc/SuSe-release" ]]; then
-    # Older SuSE/etc.
-    distro="suse"
-  elif [[ -f "/etc/redhat-release" ]]; then
-    # Older Red Hat, CentOS, etc.
-    distro="redhat"
+prepare_environment_for_werf() {
+
+  local override_prepare_environment_for_werf="$1"
+
+  [[ $override_prepare_environment_for_werf == "no" ]] && return 0
+
+  [[ $override_prepare_environment_for_werf == "auto" ]] && prompt_yes_no_skip "Need to install packages for werf?" "yes" || return 0
+
+  if ! is_command_exists git; then
+    install_package git
+  else
+    already_installed git
+  fi
+
+  if ! is_command_exists curl; then
+    install_package curl
+  else
+    already_installed curl
+  fi
+
+  if ! is_command_exists gpg; then
+    install_package gpg
+  else
+    already_installed gpg
+    echo
+  fi
```

Comment:
```suggestion
  is_command_exists gpg || install_package gpg
```

I'd really just do this and won't bother, but whatever, do the way you prefer